kernelci/build.py

```diff
@@ -321,6 +322,20 @@ def _download_file(url, dest_filename, chunk_size=1024):
     return False
 
 
+def apply_patch_mbox(path, mbox_url):
+    with tempfile.NamedTemporaryFile(prefix="kernel-patch-") as tmp_f:
+        if not _download_file(mbox_url, tmp_f.name):
+            raise FileNotFoundError(f"Error downloading patch mbox {mbox_url}")
+
+        shell_cmd("""
+set -e
+cd {path}
+git config user.name "kernelci-tsc"
+git config user.email "kernelci-tsc@groups.io"
```

Comment:
I think we should remove this and let the user define it instead.

Reply:
Made it function args with default values